package.json

```diff
@@ -32,7 +33,7 @@
     "styled-components": "^4.4.1"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "react-scripts --openssl-legacy-provider start",
```

Comment:
@jumpybuns it looks like this isn't breaking the netlify build, so that's good - but is this needed still, now that you've got your node stuff working?

Reply:
@dbridenbeck removed